src/estimation/CMakeLists.txt

```diff
@@ -16,7 +16,12 @@ set(IDYNTREE_ESTIMATION_HEADERS include/iDynTree/Estimation/BerdyHelper.h
                                 include/iDynTree/Estimation/SchmittTrigger.h
                                 include/iDynTree/Estimation/ContactStateMachine.h
                                 include/iDynTree/Estimation/BipedFootContactClassifier.h
-                                include/iDynTree/Estimation/GravityCompensationHelpers.h)
+                                include/iDynTree/Estimation/GravityCompensationHelpers.h
+                                include/iDynTree/Estimation/ExtendedKalmanFilter.h
+                                include/iDynTree/Estimation/AttitudeEstimator.h
+                                include/iDynTree/Estimation/AttitudeEstimatorUtils.h
```

Comment:
If you do not plan to use them outside iDynTree, I think we could move `AttitudeEstimatorUtils.h` in the `IDYNTREE_ESTIMATION_PRIVATE_INCLUDES` . In this way, if we need to change them we do not need to deprecated, etc, etc.